src/CoreServices/FSEvents.cs

```diff
@@ -207,8 +245,8 @@ public FSEventStream (string [] pathsToWatch, TimeSpan latency, FSEventStreamCre
 			}
 
 			var events = new FSEvent[numEvents];
-			var pathArray = new CFArray (eventPaths);
-
+			using var pathArray = new CFArray (eventPaths);
```

Comment:
To clarify to other reviewers, if the pathArray is not release after the call, we had issues with the array since it is an Array the follows the Get rule.